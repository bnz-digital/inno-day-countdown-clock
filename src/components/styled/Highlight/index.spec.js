import React from 'react'
import { shallow } from 'enzyme'

import StyledHighlight from '.'

describe('components:styled:Highlight', () => {
  it('renders <StyledHighlight/> correctly', () => {
    expect(toJson(shallow(<StyledHighlight />))).toMatchSnapshot()
  })
})
