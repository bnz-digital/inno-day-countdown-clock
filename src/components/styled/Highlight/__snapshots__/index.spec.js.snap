// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:Highlight renders <StyledHighlight/> correctly 1`] = `
.c0 {
  color: #FAA61B;
}

<span
  className="c0"
/>
`;
